code/src/components/TodoItem.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { useDispatch } from "react-redux";
+import styled from "styled-components/macro";
+
+import { Button } from "lib/Button";
+import { CustomCheckbox } from "./CustomCheckbox";
+import { todos } from "../reducers/todos";
+
+
+const ItemList = styled.li`
+  display: flex;
+  justify-content: space-between;
+  background: #fff;
+  font-size: 18px;
+  padding: 10px;
+  border: solid 2px #786253;
+  box-shadow: 8px 5px;
+  border-radius: 5px;
+  width: 320px;
+  height: auto;
+  margin: 6px 25px;
+  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
+    monospace;
+`;
+
+const TodoInputText = styled.div`
+  padding: 5px;
+`;
+
+export const TodoItem = ({ id, name, isCompleted }) => {
+  const dispatch = useDispatch();
+
+  const onTodoDelete = () => {
+    dispatch(todos.actions.removeItem(id));
+  };
+
+  return (
+    <ItemList>
+      <CustomCheckbox checked={isCompleted} />
+      <TodoInputText>{name}</TodoInputText>
+      <Button onClick={onTodoDelete}>Done it</Button>
+    </ItemList>
+  );
+};
+
+//   Might use this later, hade to mark it out before deployment.
+// const onIsCompletedChange = () => {
+//     dispatch(todos.actions.toggleComplete(id));
+//   };
```

Comment:
Since one of the requirements for this week is:
_Your app should show a count of either all todos, or all uncomplete todos (or both)_
I will mark this with requested changes :)

Reply:
Solved, now it shows both done todos and todos left! 🤓